test/leeway-build.sh

```diff
@@ -19,4 +19,5 @@ for COMPONENT in tests/components/*; do
     go test -trimpath -ldflags="-buildid= -w -s" -c -o bin/"$OUTPUT".test ./"$COMPONENT"
 done
 
-go test -trimpath -ldflags="-buildid= -w -s" -o bin/workspace -c ./tests/workspace
+echo building test tests/workspace
+go test -trimpath -ldflags="-buildid= -w -s" -o bin/workspace.test -c ./tests/workspace
```

Comment:
It seems that the workspace test had not been run before, but I didn't know why.

Reply:
ref: https://github.com/gitpod-io/gitpod/blob/30da76375c996109f243491b23e47feefab7217f/test/entrypoint.sh#L14